recipes/openjdk/all/conanfile.py

```diff
@@ -15,39 +16,49 @@ class OpenJDK(ConanFile):
     settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```

Please, add package_id() and remove both.